modules/virt-hot-plugging-disks-ui.adoc

```diff
@@ -20,16 +18,16 @@ You can make a hot plugged disk persistent so that it is permanently mounted on
 
 . Navigate to *Virtualization* -> *VirtualMachines* in the web console.
 . Select a running VM to view its details.
-. On the *VirtualMachine details* page, click *Configuration* -> *Disks*.
+. On the *VirtualMachine details* page, click *Configuration* -> *Storage*.
 
 . Add a hot plugged disk:
-.. Click *Add disk*.
+.. Click *Add*.
 .. In the *Add disk (hot plugged)* window, select the disk from the *Source* list and click *Save*.
-
+. Optional: Select the type of the interface bus. The options are VirtIO and SCSI. The default bus type is VirtIO.
```

Comment:
```suggestion
. Optional: Select the type of the interface bus. The options are *VirtIO* and *SCSI*. The default bus type is *VirtIO*.
```
Suggestion: should these be formatted as literal, if they're something being selected by the user? Not sure what this looks like in the UI so can't say for sure.